app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,33 @@
+# class LikesController < ApplicationController
+#   def create
+#     @post = Post.find(params[:post_id])
+#     @like = @post.likes.new(user: current_user)
+
+#     if @like.save
+#       redirect_to post_path(@post), notice: 'Post liked successfully.'
+#     else
+#       redirect_to post_path(@post), alert: 'Failed to like the post.'
+#     end
+#   end
+# end
```

Comment:
- [ ] Kindly remove commented lines of code. Commenting code can make it hard to read and maintain.